binderhub/tests/conftest.py

```diff
@@ -118,6 +118,23 @@ def _binderhub_config():
     return cfg
 
 
+@pytest.fixture(scope='session')
+def _binderhub_auth_config():
+    cfg = _binderhub_config()
+    cfg.BinderHub.base_url = '/services/binder/'
+    cfg.BinderHub.auth_enabled = True
+    cfg.BinderHub.use_oauth = False
+    cfg.BinderHub.use_named_servers = False
+
+    os.environ['JUPYTERHUB_API_TOKEN'] = cfg.BinderHub.hub_api_token
```

Comment:
Since the class has been imported already, setting the environment variable here is too late. Instead, we can set `cfg.HubAuthenticated.api_token = cfg.BinderHub.hub_api_token` directly. Same, I imagine, for the other environment variables. Set config values directly instead of relying on environment variables.

Reply:
I think reading https://traitlets.readthedocs.io/en/latest/config.html will help me to figure out how to do this